[2023-08-21T20:34:52.809+0000] {processor.py:156} INFO - Started process (PID=34) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:34:52.816+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:34:52.829+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:34:52.829+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:34:53.503+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:34:53.500+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:34:53.504+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:34:53.529+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.723 seconds
[2023-08-21T20:35:23.702+0000] {processor.py:156} INFO - Started process (PID=62) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:23.705+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:35:23.706+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:35:23.706+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:23.919+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:35:23.916+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:35:23.920+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:23.944+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.247 seconds
[2023-08-21T20:35:54.148+0000] {processor.py:156} INFO - Started process (PID=90) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:54.160+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:35:54.162+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:35:54.162+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:54.399+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:35:54.395+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:35:54.400+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:35:54.424+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.281 seconds
[2023-08-21T20:36:24.636+0000] {processor.py:156} INFO - Started process (PID=116) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:24.639+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:36:24.640+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:36:24.640+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:24.891+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:36:24.890+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:36:24.892+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:24.917+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.295 seconds
[2023-08-21T20:36:55.147+0000] {processor.py:156} INFO - Started process (PID=136) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:55.149+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:36:55.151+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:36:55.151+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:55.307+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:36:55.305+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:36:55.308+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:36:55.326+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.185 seconds
[2023-08-21T20:37:25.475+0000] {processor.py:156} INFO - Started process (PID=163) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:37:25.478+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:37:25.479+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:37:25.479+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:37:25.686+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:37:25.684+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:37:25.687+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:37:25.706+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.234 seconds
[2023-08-21T20:39:06.377+0000] {processor.py:156} INFO - Started process (PID=34) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:06.380+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:39:06.386+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:39:06.386+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:06.999+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:39:06.998+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:39:07.000+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:07.017+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.644 seconds
[2023-08-21T20:39:37.400+0000] {processor.py:156} INFO - Started process (PID=62) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:37.403+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:39:37.404+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:39:37.404+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:37.553+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:39:37.551+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:39:37.553+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:39:37.568+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.175 seconds
[2023-08-21T20:40:07.689+0000] {processor.py:156} INFO - Started process (PID=90) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:07.692+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:40:07.694+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:40:07.693+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:07.830+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:40:07.828+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:40:07.830+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:07.845+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.162 seconds
[2023-08-21T20:40:38.117+0000] {processor.py:156} INFO - Started process (PID=118) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:38.120+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:40:38.121+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:40:38.121+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:38.246+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:40:38.244+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:40:38.247+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:40:38.260+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.148 seconds
[2023-08-21T20:41:08.392+0000] {processor.py:156} INFO - Started process (PID=145) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:08.393+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:41:08.394+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:41:08.394+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:08.537+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:41:08.535+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:41:08.538+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:08.553+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.163 seconds
[2023-08-21T20:41:38.655+0000] {processor.py:156} INFO - Started process (PID=164) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:38.657+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:41:38.658+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:41:38.658+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:38.787+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:41:38.785+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:41:38.787+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:41:38.801+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.153 seconds
[2023-08-21T20:42:08.978+0000] {processor.py:156} INFO - Started process (PID=193) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:08.981+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:42:08.982+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:42:08.982+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:09.110+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:42:09.109+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:42:09.111+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:09.124+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.152 seconds
[2023-08-21T20:42:39.444+0000] {processor.py:156} INFO - Started process (PID=220) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:39.447+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:42:39.448+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:42:39.448+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:39.573+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:42:39.572+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:42:39.574+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:42:39.588+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.149 seconds
[2023-08-21T20:43:09.819+0000] {processor.py:156} INFO - Started process (PID=248) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:09.821+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:43:09.823+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:43:09.823+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:09.952+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:43:09.951+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:43:09.953+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:09.967+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.155 seconds
[2023-08-21T20:43:40.332+0000] {processor.py:156} INFO - Started process (PID=276) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:40.334+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:43:40.335+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:43:40.335+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:40.469+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:43:40.467+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:43:40.469+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:43:40.483+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.157 seconds
[2023-08-21T20:44:10.742+0000] {processor.py:156} INFO - Started process (PID=304) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:10.745+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:44:10.749+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:10.749+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:10.885+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:10.883+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:44:10.886+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:10.899+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.163 seconds
[2023-08-21T20:44:41.063+0000] {processor.py:156} INFO - Started process (PID=332) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:41.067+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:44:41.068+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:41.068+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:41.214+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:41.212+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/tbf_analysis_dag_v09.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tbf_analysis_dag_v09.py", line 137, in <module>
    bash_command=f"bash {GCS_TRANSFER_SCRIPT} {BUCKET}"
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 147, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 773, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 213, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'authenticate_service_acct_task' has already been added to the DAG
[2023-08-21T20:44:41.216+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:41.241+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.181 seconds
[2023-08-21T20:44:56.275+0000] {processor.py:156} INFO - Started process (PID=342) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:56.276+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:44:56.277+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.277+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:56.440+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:44:56.582+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.582+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:tbf_analysis_dag_v09
[2023-08-21T20:44:56.589+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.589+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:tbf_analysis_dag_v09
[2023-08-21T20:44:56.594+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.594+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:tbf_analysis_dag_v09
[2023-08-21T20:44:56.595+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.594+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:44:56.603+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.603+0000] {dag.py:2594} INFO - Creating ORM DAG for tbf_analysis_dag_v09
[2023-08-21T20:44:56.611+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:44:56.611+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to 2023-08-21T00:00:00+00:00, run_after=2023-08-21T00:00:00+00:00
[2023-08-21T20:44:56.625+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.354 seconds
[2023-08-21T20:45:26.897+0000] {processor.py:156} INFO - Started process (PID=375) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:26.898+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:45:26.899+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:26.899+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:27.056+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:27.078+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:27.078+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:45:27.093+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:27.093+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:45:27.105+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.211 seconds
[2023-08-21T20:45:57.274+0000] {processor.py:156} INFO - Started process (PID=403) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:57.278+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:45:57.280+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:57.280+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:57.424+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:45:57.444+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:57.444+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:45:57.459+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:45:57.458+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:45:57.471+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.203 seconds
[2023-08-21T20:46:27.651+0000] {processor.py:156} INFO - Started process (PID=432) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:27.653+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:46:27.654+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:27.654+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:27.802+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:27.825+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:27.825+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:46:27.841+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:27.841+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:46:27.855+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.207 seconds
[2023-08-21T20:46:58.077+0000] {processor.py:156} INFO - Started process (PID=451) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:58.081+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:46:58.082+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:58.082+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:58.209+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:46:58.228+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:58.227+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:46:58.242+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:46:58.242+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:46:58.253+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.184 seconds
[2023-08-21T20:47:28.359+0000] {processor.py:156} INFO - Started process (PID=480) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:28.361+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:47:28.362+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:28.362+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:28.497+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:28.519+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:28.519+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:47:28.535+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:28.535+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:47:28.547+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.190 seconds
[2023-08-21T20:47:58.867+0000] {processor.py:156} INFO - Started process (PID=508) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:58.870+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:47:58.871+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:58.871+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:59.030+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:47:59.055+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:59.054+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:47:59.072+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:47:59.072+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:47:59.084+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.220 seconds
[2023-08-21T20:48:29.334+0000] {processor.py:156} INFO - Started process (PID=536) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:29.337+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:48:29.338+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:29.338+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:29.476+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:29.495+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:29.495+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:48:29.509+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:29.509+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:48:29.520+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.192 seconds
[2023-08-21T20:48:59.580+0000] {processor.py:156} INFO - Started process (PID=563) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:59.583+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:48:59.585+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:59.585+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:59.733+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:48:59.756+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:59.755+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:48:59.772+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:48:59.771+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:48:59.784+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.210 seconds
[2023-08-21T20:49:30.081+0000] {processor.py:156} INFO - Started process (PID=591) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:49:30.083+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:49:30.084+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:49:30.084+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:49:30.205+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:49:30.224+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:49:30.223+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:49:30.237+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:49:30.237+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:49:30.250+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.171 seconds
[2023-08-21T20:50:00.365+0000] {processor.py:156} INFO - Started process (PID=619) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:00.368+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:50:00.369+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:00.369+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:00.546+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:00.576+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:00.575+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:50:00.591+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:00.591+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:50:00.603+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.242 seconds
[2023-08-21T20:50:30.762+0000] {processor.py:156} INFO - Started process (PID=637) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:30.765+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:50:30.767+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:30.767+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:30.897+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:50:30.915+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:30.915+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:50:30.929+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:50:30.929+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:50:30.940+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.185 seconds
[2023-08-21T20:51:01.077+0000] {processor.py:156} INFO - Started process (PID=665) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:01.079+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:51:01.081+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:01.081+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:01.201+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:01.219+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:01.219+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:51:01.233+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:01.233+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:51:01.243+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.172 seconds
[2023-08-21T20:51:31.440+0000] {processor.py:156} INFO - Started process (PID=692) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:31.442+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:51:31.443+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:31.443+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:31.626+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:51:31.660+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:31.659+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:51:31.692+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:51:31.692+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:51:31.705+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.272 seconds
[2023-08-21T20:52:01.957+0000] {processor.py:156} INFO - Started process (PID=719) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:01.960+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:52:01.961+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:01.961+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:02.143+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:02.165+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:02.165+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:52:02.181+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:02.181+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:52:02.191+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.239 seconds
[2023-08-21T20:52:32.364+0000] {processor.py:156} INFO - Started process (PID=748) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:32.366+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:52:32.367+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:32.367+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:32.494+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:52:32.514+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:32.513+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:52:32.528+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:52:32.528+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:52:32.538+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.178 seconds
[2023-08-21T20:53:02.708+0000] {processor.py:156} INFO - Started process (PID=776) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:02.710+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:53:02.711+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:02.711+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:02.878+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:02.903+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:02.903+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:53:02.924+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:02.924+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:53:02.940+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.236 seconds
[2023-08-21T20:53:33.177+0000] {processor.py:156} INFO - Started process (PID=805) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:33.180+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:53:33.181+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:33.181+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:33.332+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:53:33.365+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:33.365+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:53:33.382+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:53:33.382+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:53:33.402+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.228 seconds
[2023-08-21T20:54:03.697+0000] {processor.py:156} INFO - Started process (PID=824) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:03.701+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:54:03.702+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:03.702+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:03.879+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:03.902+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:03.902+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:54:03.919+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:03.918+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:54:03.930+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.242 seconds
[2023-08-21T20:54:34.207+0000] {processor.py:156} INFO - Started process (PID=851) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:34.211+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:54:34.213+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:34.213+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:34.344+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:54:34.365+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:34.364+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:54:34.380+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:54:34.379+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:54:34.391+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.190 seconds
[2023-08-21T20:55:04.603+0000] {processor.py:156} INFO - Started process (PID=879) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:04.606+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:55:04.607+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:04.607+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:04.788+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:04.812+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:04.811+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:55:04.828+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:04.828+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:55:04.842+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.243 seconds
[2023-08-21T20:55:35.137+0000] {processor.py:156} INFO - Started process (PID=907) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:35.141+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:55:35.143+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:35.143+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:35.306+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:55:35.330+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:35.330+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:55:35.346+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:55:35.346+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:55:35.357+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.232 seconds
[2023-08-21T20:56:05.431+0000] {processor.py:156} INFO - Started process (PID=935) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:05.435+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:56:05.437+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:05.437+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:05.606+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:05.629+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:05.629+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:56:05.645+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:05.645+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:56:05.657+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.234 seconds
[2023-08-21T20:56:35.864+0000] {processor.py:156} INFO - Started process (PID=963) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:35.866+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:56:35.868+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:35.867+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:36.056+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:56:36.079+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:36.078+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:56:36.095+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:56:36.095+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:56:36.108+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.249 seconds
[2023-08-21T20:57:06.352+0000] {processor.py:156} INFO - Started process (PID=989) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:06.353+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:57:06.355+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:06.355+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:06.574+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:06.616+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:06.616+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:57:06.640+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:06.640+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:57:06.657+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.310 seconds
[2023-08-21T20:57:36.998+0000] {processor.py:156} INFO - Started process (PID=1009) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:37.000+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:57:37.001+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:37.001+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:37.163+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:57:37.188+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:37.187+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:57:37.206+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:57:37.206+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:57:37.220+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.228 seconds
[2023-08-21T20:58:07.337+0000] {processor.py:156} INFO - Started process (PID=1037) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:07.339+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:58:07.341+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:07.341+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:07.519+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:07.551+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:07.550+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:58:07.573+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:07.573+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:58:07.586+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.252 seconds
[2023-08-21T20:58:37.809+0000] {processor.py:156} INFO - Started process (PID=1065) to work on /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:37.811+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/tbf_analysis_dag_v09.py for tasks to queue
[2023-08-21T20:58:37.812+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:37.812+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:37.983+0000] {processor.py:768} INFO - DAG(s) dict_keys(['tbf_analysis_dag_v09']) retrieved from /opt/airflow/dags/tbf_analysis_dag_v09.py
[2023-08-21T20:58:38.005+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:38.005+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2023-08-21T20:58:38.020+0000] {logging_mixin.py:117} INFO - [2023-08-21T20:58:38.020+0000] {dag.py:3328} INFO - Setting next_dagrun for tbf_analysis_dag_v09 to None, run_after=None
[2023-08-21T20:58:38.031+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/tbf_analysis_dag_v09.py took 0.226 seconds
